service-registry/src/main/java/io/servicecomb/serviceregistry/client/LocalServiceRegistryClientImpl.java

```diff
@@ -238,14 +240,17 @@ protected Microservice findLatest(String appId, String serviceName, VersionRule
   }
 
   @Override
-  public List<MicroserviceInstance> findServiceInstance(String selfMicroserviceId, String appId, String serviceName,
-      String strVersionRule) {
+  public MicroserviceInstanceRefresh findServiceInstance(String selfMicroserviceId, String appId, String serviceName,
```

Comment:
It's a version break change, we need to add this information to the release note.